src/structures/CommandInteractionOptionResolver.js

```diff
@@ -85,7 +85,7 @@ class CommandInteractionOptionResolver {
    * Gets an option by name and property and checks its type.
    * @param {string} name The name of the option.
    * @param {ApplicationCommandOptionType} type The type of the option.
-   * @param {string[]} properties The properties to check for for `required`.
+   * @param {string[]} properties The properties to check for `required`.
```

Comment:
This is correct as-is, doesn’t need to be changed and is also out of the scope of the PR